### ISSUE TITLE
Realtime Code Sharing

### ISSUE THREAD

**Issue by dharan1011** [google/ci_edit#154]
I was thinking a **Realtime Code Sharing** _enhancement_  will let developers share their workspace with team members and collaborate on code in real time right from their command line / terminal. _This is never done before._ A _fancy editor_ with this feature with feature will the choice for many users right ?

**Comment by aaxu** [comment]
Do you mean something similar to google docs? Is this only going to be sharing one file across multiple people for concurrent edits? If so, it seems like this would be an interesting idea, though I think it would only happen in the distant future once we polish the editor up. If it is sharing of multiple files or a directory, that seems like a separate program.. though that would be cool to work on

**Comment by dharan1011** [comment]
Yes similar to google docs. Realtime collaborating i.e sharing of file across multiple user for concurrent edit right from the terminal.
